packages/cursorless-vscode/package.json

```diff
@@ -279,6 +279,11 @@
           "default": true,
           "description": "Whether to show decorations on vscode start."
         },
+        "cursorless.commandHistory": {
```

Comment:
Shall we use the `order` field to push this one up a little bit? Or wait until we're more confident in it?

Reply:
Sounds reasonable. What would be a good order do you think?

Reply:
That setting seems completely broken to me. I have shapes and hat stability at the top
![image](https://github.com/cursorless-dev/cursorless/assets/3511326/4440bcd1-daaa-4430-99f6-b9d1086af07b)

Reply:
Try saying "cursorless settings"

Reply:
Probably first time I have used at command. Now we got the desired order, but that also means that unless you get to the settings in a particular way the order doses matter. Weird.